configure.ac

```diff
@@ -148,6 +150,15 @@ AM_CONDITIONAL([ENABLE_TCTI_MSSIM], [test "x$enable_tcti_mssim" != xno])
 AS_IF([test "x$enable_tcti_mssim" = "xyes"],
 	AC_DEFINE([TCTI_MSSIM],[1], [TCTI FOR MS SIMULATOR]))
 
+AC_ARG_ENABLE([tcti-fuzzing],
```

Comment:
Can the tcti-fuzzing coexist with other tctis? Don't we need to make sure that other tctis are disabled when fuzzing is enabled?

Reply:
AR for John add checks to configure to make sure if TCTI Fuzzing is built, it is the only TCTI being built.